x-pack/platform/plugins/shared/streams/server/routes/utils/create_connector_sse_error.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getConnectorModel,
+  isInferenceProviderError,
+  type InferenceConnector,
+} from '@kbn/inference-common';
+import { createSSEInternalError } from '@kbn/sse-utils';
+
+/**
+ * Known error message patterns that indicate context length exceeded errors from various LLM providers.
+ * This was copied from the convertUpstreamError function in the inference plugin.
```

Comment:
Could we not export it from that plugin instead since we already depend on it?

Reply:
Talked offline, we'll refactor this in another PR.